tests/pipeline/test_downsampling.py

```diff
@@ -750,14 +733,14 @@ def test_downsampled_rank(self):
         downsampled_rank = self.factor.rank().downsample('month_start')
         pipeline = Pipeline({'rank': downsampled_rank})
 
-        results_month_start = self.pipeline_engine.run_pipeline(
+        results_month_start = self.seeded_random_engine.run_pipeline(
```

Comment:
why use a seeded random loader, I want the rank value to be their sid, it makes it easier to explain. Also, given a different seed this could still pass